pyEpiabm/pyEpiabm/core/parameters.py

```diff
@@ -25,8 +38,134 @@ def __init__(self):
                                                 1.37573215, 1.563841395,
                                                 1.803041398, 2.135346254,
                                                 2.694118208, 3.964172493])
+            self.asympt_infect_icdf = np.array([0, 0.171566836, 0.424943468,
```

Comment:
Can we read these icdf arrays in from a .json file or similar - more configurable and tidier!

Reply:
Richard and I were discussing this and think this is best done in a later pull request. But I agree, definitely needs to be tidied up!